src/internal/_isThenable.js

```diff
@@ -1,6 +1,9 @@
+var _hasMethod = require('./_hasMethod');
+
+
 /**
  * Tests if a value is a thenable (promise).
  */
 module.exports = function _isThenable(value) {
-  return (value != null) && (value === Object(value)) && typeof value.then === 'function';
```

Comment:
Thats to ensure it's an `Object` and not a primitive; though it should be safe regardless to check `value.then` as long as `value != null`